content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -1,130 +1,135 @@
-# Versioning Data and Model Files
-
-DVC enables versioning large files and directories such as datasets, data
-science features, and machine learning models using Git, but without storing the
-contents in Git.
-
-This is achieved by saving information about the data in special
-[metafiles](/doc/user-guide/dvc-files-and-directories) that replace the data in
-the repository. These can be versioned with regular Git workflows (branches,
-pull requests, etc.)
-
-To actually store the data, DVC uses a built-in <abbr>cache</abbr>, and supports
-synchronizing it with various types of
-[remote storage](/doc/command-reference/remote). This allows for easy data and
-model versioning, storage, and sharing — right alongside code.
-
-![](/img/model-versioning-diagram.png) _Code and data flows in DVC_
-
-In this basic use case, DVC is a better alternative to
-[Git-LFS / Git-annex](/doc/user-guide/related-technologies) and to ad-hoc
-scripts used to manage ML <abbr>artifacts</abbr> (training data, models, etc.)
-on cloud storage. DVC doesn't require special services, and works with
-on-premises storage (e.g. SSH, NAS) as well as any major cloud storage provider
-(Amazon S3, Microsoft Azure, Google Drive,
-[among others](/doc/command-reference/remote/add#supported-storage-types)).
-
-> For hands-on experience, we recommend following the
-> [versioning tutorial](/doc/use-cases/versioning-data-and-model-files).
-
-## DVC is not Git!
-
-DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to track
-data files and directories for versioning (among other purposes). They point to
-specific data contents in the <abbr>cache</abbr>, providing the ability to store
-multiple data versions out-of-the-box.
-
-Full-fledged
-[version control](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control)
-is left for Git and its hosting platforms (e.g. GitHub, GitLab) to handle. These
-are designed for source code management (SCM) however, and thus ill-equipped to
-support data science needs. That's where DVC comes in: with its built-in data
-<abbr>cache</abbr>, reproducible [pipelines](/doc/start/data-pipelines), among
-several other novel features (see [Get Started](/doc/start/) for a primer.)
-
-## Track data and models for versioning
+# Versioning Data and Models
+
+Data science teams today face data management questions around versioning
+datasets, data artifacts, and machine learning models. How do we keep track of
+changes in data, code, and ML models? What’s the best way to organize and store
+multiple versions of data files for safe, persistent access? How can the
+lifecycle of data and models be followed and enforced?
+
+![](/img/data_ver_complex.png) _Exponential complexity of DS projects_
```

Comment:
New draft figure idea

![image](https://user-images.githubusercontent.com/1477535/97093415-57e7db80-1611-11eb-8c79-84e7be7f630d.png)

Reply:
Although it would probably be better to represent the different lifecycles of code/data/models. A bit like figures from https://martinfowler.com/articles/cd4ml.html e.g.:

![image](https://user-images.githubusercontent.com/1477535/97093447-c167ea00-1611-11eb-96b6-6edc89c10e05.png)